app/models/crystal_set.rb

```diff
@@ -2,4 +2,8 @@
 
 class CrystalSet < ApplicationRecord
   has_many :crystals
+
+  def crystal_count
+    self.crystals.count
+  end
```

Comment:
Love this method, I used something a lot more annoying. Might steal it with your permission.

Reply:
Open source! What's mine is yours!